cli/src/pcluster/models/cluster_resources.py

```diff
@@ -74,6 +74,22 @@ def batch_compute_environment(self):
         """Return Batch compute environment."""
         return self._get_output("BatchComputeEnvironmentArn")
 
+    def get_failure_reason(self):
+        """Reason of the failure when the cluster_status is in CREATE_FAILED."""
+        stack_events = get_all_stack_events(self.name)
+        wait_conditiaon_failure_events = None
+        for chuck in stack_events:
```

Comment:
Is `chuck` meant to be `chunk`? I do like `chuck` better though.

Reply:
oops, fix